libuavcan/src/transport/uc_can_acceptance_filter_configurator.cpp

```diff
@@ -16,24 +16,22 @@ const unsigned CanAcceptanceFilterConfigurator::DefaultAnonMsgID;
 
 int16_t CanAcceptanceFilterConfigurator::loadInputConfiguration(AnonymousMessages load_mode)
 {
-    multiset_configs_.clear();
```

Comment:
This logic is highly complicated. One would expect that the result of `loadInputConfiguration()` would be having the config multiset loaded with unoptimized set of filters. The fact that the multiset is not getting cleared here is highly counter-intuitive. As for `applyConfiguration()`, see my comment there.